src/craterstats/cli.py

```diff
@@ -306,5 +310,11 @@ def main(args0):
         if f in {'txt'}:
             cps.create_summary_table()
 
-if __name__ == '__main__':
-    main(None)
\ No newline at end of file
+def main_default():
```

Comment:
This seems unwieldy (and you mis-spelled this function the one time you use it).  Why not just set the *arg0* argument on `main()` to default to None?  That seems cleaner, and then setup.py can just point to `main`.

Reply:
oh good catch, that will be much cleaner